sig-and-wg/sig-core/proposals/generating-changelog-with-pr-labels.md

```diff
@@ -0,0 +1,105 @@
+# Generating changelog with pull request labels
+
+Created on 2018-08-06 by Paweł Kosiec (@pkosiec).
+
+## Status
+
+Proposed on 2018-08-06.
+
+## Key assumptions
+
+- Open source community wants as much transparency as possible.
+- Changelog should be easy to read
+- Changelog should be available for developers with offline access, if they have cloned repository
+- Defining changes by contributors should be as easy as possible.
+
+## Proposed solution
+
+In one sentence: Categorizing changes with pull request labels.
+
+### Workflow
+
+- All developers should categorize their changes with PR labels
+- All changes with specific changelog-related labels are included in changelog
+- Pull requests should have a title written in imperative mood (like commit messages). The title will be included in changelog, along with the author and a link to the particular PR.
+- No label on a pull request means that the change won't be included in changelog.
+
+### Result
+
+- The changelog is written to CHANGELOG.md file and put in GitHub release description via [GitHub API](https://developer.github.com/v3/repos/releases/#create-a-release) for a specific git tags
+- All changes are grouped by selected labels
+
+Desired output would look like this:
+    - [example GitHub releases](https://github.com/lerna/lerna-changelog/releases)
+    - [example CHANGELOG.md](https://github.com/lerna/lerna-changelog/blob/master/CHANGELOG.md)
+
+### Labels
+
+Every label listed below is taken into account during changelog generation process.
```

Comment:
This seems a bit inconsistent with the next paragraph. I would propose removing this sentence and leaving the paragraph below.